packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -1011,16 +999,8 @@ class _ActionSheetMainSheetState extends State<_ActionSheetMainSheet> {
 
   @override
   Widget build(BuildContext context) {
-    // The layout rule:
-    //
-    // 1. If there are <= 3 buttons and a cancel button, or 1 button without a
-    //    cancel button, then the actions section should never scroll.
-    // 2. Otherwise, then the content section takes priority to take over spaces
-    //    but must leave at least `actionsMinHeight` for the actions section.
-    final int numActions = widget.actions?.length ?? 0;
-    final bool actionsMightScroll =
-        (numActions > 3 && widget.hasCancelButton) ||
-        (numActions > 1 && !widget.hasCancelButton) ;
+    // The content section takes priority for vertical space but must leave at
+    // least `actionsMinHeight` for the actions section.
```

Comment:
this should be _kActionSheetActionsSectionMinHeight?

Reply:
Yes! Thanks for pointing out. Fixed.